src/components/NcAvatar/NcAvatar.vue

```diff
@@ -474,7 +474,7 @@ export default {
 		},
 		tooltip() {
 			if (this.disableTooltip) {
-				return false
+				return null
```

Comment:
Would make `undefined` not make more sense?

Reply:
I have no preference here tbh. Both works. If you prefer `undefined`, that's fine with me.

Reply:
No just a question :)